Gemfile

```diff
@@ -11,13 +11,13 @@ git_source(:github) do |repo_name|
 end
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails"
-gem "rails", "6.0.3.6"
+gem "rails", "6.1.3.1"
 
 # Improve boot time
 gem "bootsnap", require: false
 
 # Database
-gem "composite_primary_keys"
+gem "composite_primary_keys", git: "https://github.com/composite-primary-keys/composite_primary_keys.git" # unreleased rails 6.1 support
```

Comment:
Are we OK with having an unreleased gem or do we need to wait for an official release?

Reply:
We've had an unreleased gem in the past while we wait for an official release. I don't know that we can wait as I need rails 6.1 to continue some of my work downstream.